Add nginx adjustments to haproxy config requirements.

SimpleTechGuy opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/amidaware/trmm-docs?pullRequest=311) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/amidaware/trmm-docs?pullRequest=311) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/amidaware/trmm-docs?pullRequest=311) it.</sub>

SimpleTechGuy reviewed (commented):
yikes, looking back maybe I should have added the nginx adjustments before line 472?

SimpleTechGuy commented:
> yikes, looking back maybe I should have added the nginx adjustments before line 472?

added a new commit, moved nginx adjustments to appear beneath meshcentral adjustments.
